props-core/src/test/java/fj/data/fingertrees/FingerTreeTest.java

```diff
@@ -36,33 +34,71 @@ void validateSize(List<Integer> list) {
         assertThat(ft.length(), equalTo(list.length()));
     }
 
-    @Test
-    public void testSeqString() {
+    public FingerTree<Integer, Integer> midSeq() {
         FingerTree<Integer, Integer> ft = FingerTree.emptyIntAddition();
-//        out.println(ft.toString());
-        String actual = List.range(1, 10).foldLeft(ft2 -> i -> {
+        //        out.println(ft.toString());
+        return List.range(1, 10).foldLeft(ft2 -> i -> {
             FingerTree<Integer, Integer> ft3 = ft2.snoc(i);
 //            out.println(ft3.toString());
             return ft3;
-        }, ft).toString();
+        }, ft);
+    }
+
+
+
+    @Test
+    public void testSeqString() {
+//        FingerTree<Integer, Integer> ft = FingerTree.emptyIntAddition();
+//        out.println(ft.toString());
+        String actual = midSeq().toString();
         String expected = "Deep(9 -> One(1 -> 1), Deep(6 -> One(3 -> Node3(3 -> V3(2,3,4))), Empty(), One(3 -> Node3(3 -> V3(5,6,7)))), Two(2 -> V2(8,9)))";
         assertThat(actual, equalTo(expected));
     }
 
-    @Test
-    public void testQueueString() {
+    public FingerTree<Integer, P2<Integer, Integer>> midPriorityQueue() {
         FingerTree<Integer, P2<Integer, Integer>> ft = FingerTree.emptyIntMax();
 //        out.println(ft.toString());
-        String actual = List.range(1, 10).foldLeft(ft2 -> i -> {
+        return List.range(1, 10).foldLeft(ft2 -> i -> {
             int j = i % 2 == 0 ? 2 * i : i;
             FingerTree<Integer, P2<Integer, Integer>> ft3 = ft2.snoc(P.p(j, j));
 //            out.println(ft3.toString());
             return ft3;
-        }, ft).toString();
+        }, ft);
+    }
+
+    @Test
+    public void testQueueString() {
+//        FingerTree<Integer, P2<Integer, Integer>> ft = FingerTree.emptyIntMax();
+//        out.println(ft.toString());
+        String actual = midPriorityQueue().toString();
         String expected = "Deep(16 -> One(1 -> (1,1)), Deep(12 -> One(8 -> Node3(8 -> V3((4,4),(3,3),(8,8)))), Empty(), One(12 -> Node3(12 -> V3((5,5),(12,12),(7,7))))), Two(16 -> V2((16,16),(9,9))))";
         assertThat(actual, equalTo(expected));
 
     }
 
+    @Test
+    public void stream() {
+        FingerTree<Integer, Integer> ft = midSeq();
+        out.println(ft);
+        Stream<Integer> s = ft.toStream();
+        System.out.println(s.toList());
+
+        out.println(midPriorityQueue());
```

Comment:
missing asserts?
(same for ```split```)
Also commented code and ```println``` calls should probably be removed.